kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvServiceProvider.java

```diff
@@ -163,48 +181,68 @@ private Service transform(IVmOpenApiService ptvService) {
     
     return service;
   }
-  
-  private List<LocalizedValue> getLocalizedValues(String type, List<VmOpenApiLocalizedListItem> items) {
-    if (items != null && !items.isEmpty()) {
-      List<LocalizedValue> result = new ArrayList<>();
-      List<VmOpenApiLocalizedListItem> typeItems = new ArrayList<>();
-      
-      for (VmOpenApiLocalizedListItem item : items) {
-        if (type.equals(item.getType())) {
-          typeItems.add(item);
+
+  private void parseChannelIds(IVmOpenApiService ptvService, Service service) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "ptvService". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)